featuretools/computational_backends/utils.py

```diff
@@ -50,19 +48,20 @@ def wrapped(*args, **kwargs):
     return inner_decorator
 
 
-def datetime_round(dt, freq, round_up=False):
+def datetime_round(dt, freq):
     """
-    Taken from comments on the Pandas source: https://github.com/pandas-dev/pandas/issues/4314
-
     round down Timestamp series to a specified freq
     """
-    if round_up:
-        round_f = np.ceil
+    if not freq.is_absolute():
+        raise ValueError("Unit is relative")
+
+    if freq.unit == 'm':
+        unit = 't'
     else:
-        round_f = np.floor
-    dt = pd.DatetimeIndex(dt)
-    freq = to_offset(freq).delta.value
-    return pd.DatetimeIndex(((round_f(dt.asi8 / freq)) * freq).astype(np.int64))
+        unit = freq.unit
+
+    freq = str(freq.value) + unit
+    return dt.apply(lambda x: x.floor(freq))
```

Comment:
we should confirm this gives the same results for approximate as before on full integration test suite.